CHANGELOG/CHANGELOG-1.13.md

```diff
@@ -0,0 +1,88 @@
+- Containerize Windows userspace OVS processes and run them in a container of the Antrea Agent Pod to align with the Linux design. ([#4936](https://github.com/antrea-io/antrea/pull/4936) [#5052](https://github.com/antrea-io/antrea/pull/5052), [@rajnkamr] [@Atish-iaf])
+- Add a new option `ContainerRuntime` to allow user to configure the container runtime while using the script `Prepare-Node.ps1` on K8s Windows Node. ([#5071](https://github.com/antrea-io/antrea/pull/5071), [@NamanAg30])
+- Add support of TLS, HTTP, and HTTPS protocols for FlowAggregator to connect to the ClickHouse DB, and allow users to specify the CA certificate for TLS and HTTPS. ([#5171](https://github.com/antrea-io/antrea/pull/5171), [@yuntanghsu])
+- Enhance Antrea L7 NetworkPolicy to support the TLS protocol. ([#4932](https://github.com/antrea-io/antrea/pull/4932), [@hongliangl])
+- Add command `antctl upgrade api-storage` to support resource storage version migration for Antrea CRDs. ([#5198](https://github.com/antrea-io/antrea/pull/5198), [@hongliangl])
+- Add support for removing the associated stale conntrack entries when UDP Endpoints are removed, with which UDP requests can be redirected to other Endpoints immediately rather than waiting for the conntrack entries to expire. ([#5112](https://github.com/antrea-io/antrea/pull/5112), [@hongliangl])
+- Add Egress information to flow records for Pod-to-external flows in FlowExporter. ([#5088](https://github.com/antrea-io/antrea/pull/5088), [@dreamtalen])
+- Increase accuracy of Pod information in the flow records by adding a Pod store in FlowExporter and FlowAggregator for them to fetch the Pod information. ([#5185](https://github.com/antrea-io/antrea/pull/5185), [@yuntanghsu])
+- Add support for Service annotation `service.kubernetes.io/topology-mode` in AntreaProxy since the old `service.kubernetes.io/topology-aware-hints` annotation has been deprecated in Kubernetes 1.27. ([#5241](https://github.com/antrea-io/antrea/pull/5241), [@mengdie-song])
+- Support the `service.kubernetes.io/service-proxy-name` label in AntreaProxy to align with KEP 2447. ([#4973](https://github.com/antrea-io/antrea/pull/4973), [@hongliangl])
+- Add a new Prometheus metric to represent the number of packets dropped by OVS meter. ([#5165](https://github.com/antrea-io/antrea/pull/5165), [@mengdie-song])
+- Add support for the `sort-by` flag in more `antctl get` commands for more fields. ([#4346](https://github.com/antrea-io/antrea/pull/4346), [@jainpulkit22])
+- Add the `kubeAPIServerOverride` option to allow user to override the kube-apiserver address for antrea-controller. ([#5056](https://github.com/antrea-io/antrea/pull/5056), [@tnqn])
+- Add documentation for deploying Antrea with a Rancher cluster. ([#4733](https://github.com/antrea-io/antrea/pull/4733), [@jainpulkit22])
+
+### Changed
+
+- Multiple APIs are promoted from alpha to beta.
```

Comment:
We should also mention the alpha versions are deprecated and will be removed in a future release.

Reply:
Added.